docs/source/UserGuide/Tutorials/JobManagementAdvanced/index.rst

```diff
@@ -447,11 +447,19 @@ Jobs that can (or should) run using more than 1 processor should be described as
       j.setExecutable('ls',arguments='-l')
       j.setExecutable('echo', arguments='hello again')
       j.setName('MP test')
+      j.setTag('MultiProcessor')
```

Comment:
Shouldn't this be, e.g., "16Processors", because the "MultiProcessor" is added automatically?

Reply:
It's added automatically from v6r20p5 only, so I kept the original explanation.

Reply:
But still one should add number of processors as a tag, right?

Reply:
We have to add one more function setNumberOfProcessors() with an int argument, similar to setCPUTime()

Reply:
Should this comment be turned into an issue?

Reply:
#4127